django_tiptap/templates/forms/tiptap_textarea.html

```diff
@@ -817,26 +841,115 @@
 
     let showingMenubar = false
 
+    const headingLevels = []
+    // {% if 'h1' in widget.config.extensions  %}
```

Comment:
Did you mean to commit all these comments?

Reply:
they're not actually comments, they're a part of django templating :slightly_smiling_face: , but since javascript doesn't recognise it, it was showing me constant red underlines and I didn't like it, so I commented it out

Reply:
Ah yeah, makes sense. The templating doesnt care about comments and if it makes the IDE happy then why not :)